### PR TITLE
Revert encoding null values to 0x0602

### PR DESCRIPTION
**Pull request by staltz**
This PR is ready, but is pending on *design approval* from multiple stakeholders.

### CODE REVIEW

**Comment by cryptix** [comment]
for future reference the discussion is here: https://github.com/ssb-ngi-pointer/ssb-binary-field-encodings-spec/issues/4#issuecomment-879746796

**Comment by staltz** [comment]
This PR is now ready for review now that https://github.com/ssb-ngi-pointer/ssb-binary-field-encodings-spec/pull/6 has been merged

**Comment by cryptix** [comment]
We should also change how the previous on message:1 is encoded, right?

File: internal/bencodeext/tanglepoint.go [hunk 1]
```diff
@@ -49,7 +41,7 @@ func (tp TanglePoint) MarshalBencode() ([]byte, error) {
 	}
 
 	if n := len(tp.Previous); n == 0 {
-		m["previous"] = zeroBendyMessage
+		m["previous"] = Null
```
Comment (staltz) [inline internal/bencodeext/tanglepoint.go:44]: @cryptix You mean here? It's supposed to be `[]byte{0x06, 0x02}`?

Comment (cryptix) [reply internal/bencodeext/tanglepoint.go:44]: no, those are just the tangles.

I mean the encoding of previous on the first message on a feed.

**Comment by staltz** [comment]
Okay, you were much quicker to identify which files need to be changed, of course. Thanks for doing it! The only reason why I made this PR was to take some of the work off your shoulders after changing the spec. I hope I didn't bother.

**Comment by cryptix** [comment]
No bother at all. Thanks for getting the ball rolling!

**Pull request merged**